target/linux/apm821xx/image/sata.mk

```diff
@@ -9,7 +9,7 @@ define Device/wd_mybooklive
   DEVICE_ALT0_VENDOR := Western Digital
   DEVICE_ALT0_MODEL := My Book Live Duo
   DEVICE_PACKAGES := kmod-usb-dwc2 kmod-ata-dwc kmod-usb-ledtrig-usbport \
-	kmod-usb-storage kmod-fs-vfat wpad-basic-mbedtls
+	kmod-usb-storage kmod-fs-vfat
```

Comment:
Perhaps `$(PACKAGE_NO_WIRELESS)` fits here too? MBL and MBLD are wired devices.

Reply:
Thanks for the review, [should be fixed](https://github.com/openwrt/openwrt/pull/12917/commits/97d2b66ba1de921b9c602b1dcfbd212e68d0a90c).